app/src/main/kotlin/no/nav/tiltakspenger/vedtak/repository/behandling/PostgresBehandlingRepo.kt

```diff
@@ -33,7 +33,7 @@ internal class PostgresBehandlingRepo(
     private val søknadDAO: SøknadDAO = SøknadDAO(),
     private val tiltakDAO: TiltakDAO = TiltakDAO(),
     private val utfallsperiodeDAO: UtfallsperiodeDAO = UtfallsperiodeDAO(),
-) : BehandlingRepo {
+) : BehandlingRepo, BehandlingDAO {
```

Comment:
Arver denne både fra `BehandlingRepo` og `BehandlingDAO`? Det har jeg ikke sett før!
Men `BehandlingRepo` og `BehandlingDAO` høres ut som to ting som prøver litt på å være det samme.
Stemmer det? Kunne de blitt slått sammen? For synes det virker litt rart med den BehandlingDAO-en her egentlig.